pkg/network/master/egressip.go

```diff
@@ -25,6 +27,7 @@ type egressIPManager struct {
 	tracker            *common.EgressIPTracker
 	networkClient      networkclient.Interface
 	hostSubnetInformer networkinformers.HostSubnetInformer
+	nodeInformer       kcoreinformers.NodeInformer
```

Comment:
this has to been initialized, plumbed down from the master initialization

Reply:
This seems actually to be initalized in master, just check https://github.com/openshift/sdn/blob/master/pkg/network/master/master.go#L37. I'm wondering whether it'd be passed to the start function as well.

Reply:
you are creating a new object that is using an informer of that type

```go
func newEgressIPManager() *egressIPManager {
	eim := &egressIPManager{}
	eim.tracker = common.NewEgressIPTracker(eim)
	return eim
}
func (eim *egressIPManager) Start(networkClient networkclient.Interface, hostSubnetInformer networkinformers.HostSubnetInformer, netNamespaceInformer networkinformers.NetNamespaceInformer) {
	eim.networkClient = networkClient
	eim.hostSubnetInformer = hostSubnetInformer
	eim.tracker.Start(hostSubnetInformer, netNamespaceInformer)
}
```
, but you are not cascading it down from the master
https://github.com/openshift/sdn/blob/578df042fa70429b3f40765513ee485f0dc49d60/pkg/network/master/master.go#L117-L118